torch/ao/quantization/fx/prepare.py

```diff
@@ -174,35 +174,37 @@ def is_input_arg_dtype_supported_by_backend(
     is_bias = node_arg_is_bias(node, arg, backend_config)
     is_activation = not is_weight and not is_bias
     if is_activation:
-        is_dynamic = dtype_config.is_dynamic
-        if is_dynamic:
-            input_activation_dtype = dtype_config.input_dtype
-            # TODO: change this after the is_dynamic refactor is landed
-            compute_dtype = node_name_to_target_dtype[node.name].get("input_activation_compute_dtype", None)
-            return input_activation_dtype is None or \
-                compute_dtype == input_activation_dtype
+        qconfig_info = node_name_to_target_dtype[node.name].get(
+            "input_activation_dtype")
+        if qconfig_info is not None:
+            qconfig_dtype, qconfig_is_dynamic = qconfig_info
         else:
-            input_activation_dtype = dtype_config.input_dtype
-            return input_activation_dtype is None or \
-                node_name_to_target_dtype[node.name]["input_activation_dtype"] == input_activation_dtype
+            qconfig_dtype, qconfig_is_dynamic = None, None
+        # TODO(future PR): remove the cast to bool below after figuring
+        # out why backend_config has is_dynamic set to None in some cases.
```

Comment:
probably because it's initialized to None, I feel we can initialize it to False

Reply:
tried that, didn't fully solve the issue.  Let's do in a separate PR.